docs/usage/types/custom.md

```diff
@@ -321,10 +321,95 @@ except ValidationError as exc:
     """
 ```
 
-## Handling third-party types
+## Using `__get_pydantic_core_schema__`
```

Comment:
I feel like the section title should not mention the "__get_pydantic_core_schema__". It's the first time it's introduced in the document, and the reader is unaware of its meaning.

Some options if you agree with what I said:
- Implement the Pydantic protocol
- Create your own Pydantic Type
- Use Pydantic protocol for your types
- ?

If you don't agree, just resolve the thread. 👀

Reply:
In the linear issue @dmontagu specifically asked for that title with the function name in it. I’m going to merge as is and we can discuss more later, it’s just docs

Reply:
I would prefer a heading in the format "Customizing JSON schema with `__get_pydantic_core_schema__`" or similar. It provides the usage context and uses the relevant keyword. Will show up well in doc search.

Reply:
```suggestion
## Customizing validation with `__get_pydantic_core_schema__`
```